plugins/systemair/__init__.py

```diff
@@ -53,6 +55,7 @@ def init_serial_connection(self, serialport, slave_address):
             return False
 
     def _read_modbus(self):
+        self._lockmb.acquire()
```

Comment:
according to python [documentation](https://docs.python.org/3.4/tutorial/errors.html#defining-clean-up-actions):

>  The finally clause is also executed “on the way out” when any other clause of the try statement is left via a break, continue or return statement.


So i think that lock will be released anyway. I dont know exact purpose of `init_serial_connection()`, it's not my piece of code.